src/main/java/frc/robot/commands/SlapstickCommand.java

```diff
@@ -0,0 +1,28 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.SlapstickSubsystem;
+
+public class SlapstickCommand extends CommandBase {
```

Comment:
I would get in the habit to stop making commands like this. This is a lot of boilerplate code that could be put into a 3-line method on the SlapStickSubsystem, very similar to what we're doing in the ArmSubsystem. Code would probably look like:

```java
public Command setSlapstickPosition(boolean open) {
  return new InstantCommand(() -> setState(open), this);
}
```

Overall, our default should be to make commands like the above (called "Functional" commands) instead of creating entire objects. There may be cases where we need to make an entire object, but that should be our last choice.